x-pack/solutions/security/plugins/session_view/public/components/process_tree_node/use_button_styles.ts

```diff
@@ -6,12 +6,11 @@
  */
 
 import { useMemo } from 'react';
-import { transparentize } from '@elastic/eui';
 import { CSSObject } from '@emotion/react';
 import { useEuiTheme } from '../../hooks';
 
 export const useButtonStyles = () => {
-  const { euiTheme, euiVars } = useEuiTheme();
+  const { euiTheme } = useEuiTheme();
```

Comment:
in this file I tried replacing vis colors with color tokens, but it requires alignment with designers as there is no match to the old vis colors